lua/telescope/_extensions/yabs.lua

```diff
@@ -0,0 +1,49 @@
+local telescope = require('telescope')
+local actions = require('telescope.actions')
+local pickers = require('telescope.pickers')
+local finders = require('telescope.finders')
+local sorters = require('telescope.sorters')
+local Yabs = require('yabs')
+
+local function select_task(tasks_function, opts)
+  pickers.new(opts, {
+    prompt_title = 'Select a session',
```

Comment:
Shouldn't this be `'Select a task'`, or similar?

Reply:
Oh, right, just a copy-pase :D